File: C3-Assignments/C3-03/04.cpp
//04. Write a C++ program to print first N odd natural number in reverse order
#include<iostream>
using namespace std;
int main()
{
    int n, i;
    cout<< "Enter a number: ";
    cin>>n;
    n*=2;
    for(i=1;i<=n;i++)
    {
        if(i%2!=0)
            cout<< i << " ";
    }
    return 0;
}

File: Python/PY 25 AS 11 more on while/04.py
#   04. Write a python script to calculate sum of first N odd natural numbers.
n = int(input("Enter a number: "));
print("Sum of first %d Odd natural number is: "%(n),end="");
sum = 0;
while n:
    sum = sum + 2*n-1;
    n-=1;
print(sum);